travel-planner/travel-planner-ui/src/Components/SelectDirections/SelectDirections.jsx

```diff
@@ -2,13 +2,15 @@ import "./SelectDirections.css";
 import React, { useState, useEffect } from "react";
 import axios from "axios";
 import Dropdown from "react-bootstrap/Dropdown";
-import CustomToggle from "./../Dropdown/CustomToggle"
-import CustomMenu from "./../Dropdown/CustomMenu"
+import CustomToggle from "./../Dropdown/CustomToggle";
+import CustomMenu from "./../Dropdown/CustomMenu";
+import DirectionReasonBubble from "./DirectionReasonBubble/DirectionReasonBubble";
 
 export default function SelectDirections({ PORT, mapId }) {
 	// get all directions from backend
 	const [directions, setDirections] = useState(null);
 	const [curDirections, setCurDirections] = useState(null);
+	const [openMoreInfoBubble, setOpenMoreInfoBubble] = useState(false);
```

Comment:
this variable name is a bit funky, usually booleans phrase a question, like: 'isInfoBubbleShown'